vtysh/vtysh_main.c

```diff
@@ -351,6 +422,7 @@ int main(int argc, char **argv, char **env)
 	const char *histfile = NULL;
 	const char *histfile_env = getenv("VTYSH_HISTFILE");
 	const char *logpath = getenv("VTYSH_LOG");
+	int exec_timeout = 0;
```

Comment:
I think we should also support loading this from environment, which might just be a two-liner here (`if (getenv("VTYSH_TIMEOUT")) exec_timeout = atoi(getenv("VTYSH_TIMEOUT"));`)

Reply:
yes, sure, I'll make that change